From e3f3e55bd45516e57bd40bcd12798f8287546b07 Mon Sep 17 00:00:00 2001
From: Petro Ladkin <petro@auxoft.com>
Date: Thu, 8 Dec 2016 18:06:00 +0200
Subject: [PATCH] ### temp

---
 bin/af-hotfix | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/af-hotfix b/bin/af-hotfix
index b71bdfd..67d1213 100644
--- a/bin/af-hotfix
+++ b/bin/af-hotfix
@@ -161,7 +161,7 @@ cmd_finish() {
 
     mssg "start feature $PATCH_FEATURE_NAME"
 
-    $AF_DIR/af feature start "$PATCH_FEATURE_NAME" || \
+    $AF_DIR/af feature start "$PATCH_FEATURE_NAME" -s || \
       die "Could not start feature $PATCH_FEATURE_NAME."
 
 
@@ -187,7 +187,7 @@ cmd_finish() {
 
         git_do commit -a -m "### apply patch $APPLIED_PATCH_FILE"
 
-        $AF_DIR/af feature finish || \
+        $AF_DIR/af feature finish -s || \
           die "Could not finish feature $PATCH_FEATURE_NAME."
       fi
 
-- 
2.9.3 (Apple Git-75)

