From a41cf1db68a4612e410554cb5bc0f30b3cc57710 Mon Sep 17 00:00:00 2001
From: Petro Ladkin <petro@auxoft.com>
Date: Thu, 8 Dec 2016 18:09:56 +0200
Subject: [PATCH] ### test

---
 bin/af-hotfix | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/af-hotfix b/bin/af-hotfix
index 7e09fd6..8849e56 100644
--- a/bin/af-hotfix
+++ b/bin/af-hotfix
@@ -165,7 +165,7 @@ cmd_finish() {
 
     mssg "start feature $PATCH_FEATURE_NAME"
 
-    $AF_DIR/af feature start "$PATCH_FEATURE_NAME" -s || \
+    $AF_DIR/af -s feature start "$PATCH_FEATURE_NAME" || \
       die "Could not start feature $PATCH_FEATURE_NAME."
 
 
@@ -191,7 +191,7 @@ cmd_finish() {
 
         git_do commit -a -m "### apply patch $APPLIED_PATCH_FILE"
 
-        $AF_DIR/af feature finish -s || \
+        $AF_DIR/af -s feature finish || \
           die "Could not finish feature $PATCH_FEATURE_NAME."
       fi
 
-- 
2.9.3 (Apple Git-75)

